common/scripts/base.sh

```diff
@@ -139,3 +138,25 @@ function showUsageAndExitDefault() {
   echoBold "Ex: ./deploy.sh"
   exit 1
 }
+
+function deploy_common_service()
```

Comment:
@ravihansa3000: yes, currently we are only deploying common services (marathon-lb, wso2 shared dbs - gov db and user db) using deploy_common_services().
**function deploy_common_services() {
  deploy_common_service  'marathon-lb'
  deploy_common_service 'wso2-shared-dbs'
}**

And we are using a separate function 'deploy_common_service' to avoid the below code repetition in that function:
**function deploy_common_service()
{
  if ! bash ${mesos_artifacts_home}/common/${1}/deploy.sh; then
    echoError "Non-zero exit code returned when deploying ${1}"
    exit 1
  fi
}**

I think this is the correct approach. WDYT?